### DISCUSSION TITLE
Remove some extension/tests/etc like -> "32bit build deprecated, and will be removed in next release axmol-2.2.0"

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#1602] [General]
Some canditates be:
- extensions/GUI: https://github.com/axmolengine/axmol/tree/dev/extensions/GUI
   - its deprecrated => better using Node: UI
   
- extensions/DragonBones:  https://github.com/axmolengine/axmol/tree/dev/extensions/DragonBones
   - original link no longer exists: https://[www.dragonbones.com](https://www.dragonbones.com/)/ 
   - We can try to make is working again with axmol but if the editor not existing longer...for what?

- Zwoptex
    - Original page: https://zwopple.com/zwoptex page: 404 Not Found
    https://github.com/axmolengine/axmol/tree/dev/tests/cpp-tests/Source/ZwoptexTest

**Comment by aismann** [comment]
BTW:
Add all new extensions to the readme (mostly for correct license info)

@halx99 
Can I start all stuffs for add/and set deprecrated (removing with axmol 2.20)?

**Reply by halx99** [reply]
yes, you can

**Reply by aismann** [reply]
@halx99 ,
I have tried to replace all GUI stuffs in ccp-tests, lua-tests.
This is lot of refactoring and without big improvements.
Maybe removing of the extension 'GUI" is "not realy a good idea" because of using in the tests projects.

**Reply by halx99** [reply]
OK

**Reply by aismann** [reply]
Maybe an info like "deprecrated (only internal used...) " is better.

**Comment by aismann** [comment]
Closed it see PR #1604
